geonotebook/wrappers/raster.py

```diff
@@ -8,6 +8,39 @@
 
 from shapely.geometry import Polygon
 
+class TMSRasterData(object):
+    """
+    A wrapper for a TMS server.
+
+    The underlying TMS server must carry the 'port', 'host', and 'url_pattern'
+    properties as well as the 'bind' and 'unbind' methods.  The bind method will
+    assign a random port at the specified host if the 'port' argument is unset.
+    """
+    def __init__(self, tms, name=None):
+        self.tms = tms
```

Comment:
I'm wondering if this tms object is something that can be created inside a reader rather than passed in directly  to the TMSRasterData object?

The original goal was to provide 'uri' style syntax to RasterData() so we could reference a bunch of different data sources (e.g. files,  postgis databases,  exotic filesystems etc). That maybe be a pipe dream,  but maybe you can point me towards where/how this tms object is actually being instantiated?